bin/chat-fastapi.py

```diff
@@ -69,18 +88,14 @@ async def verify_captcha_middleware(request: Request, call_next):
 
 # Serve the CAPTCHA verification page (basic HTML form)
 @app.get(f"{CHAINLIT_URI}/verify_captcha_page")
-async def captcha_page(request: Request):
-
-    host = request.headers.get('X-Forwarded-Host', request.headers.get('Host'))
-    form_action = f"https://{host}{CHAINLIT_URI}/verify_captcha"
```

Comment:
reverted this since the new HTTPS middleware redirect should handle this

Reply:
true